test/controllers/helpers/htmlAssert.helper.ts

```diff
@@ -21,6 +21,10 @@ export default class HtmlAssertHelper {
     return this.getAttributeValue(selector, 'value')
   }
 
+  public hasClass(selector: string, attr: string): boolean {
+    return this.getElement(selector)?.className === attr
```

Comment:
can className be an array?

Reply:
Nope. It'll always be a single string of whatever the value is in the `class` property of the element, e.g. `govuk-tag` or `govuk-tag govuk-tag--grey`

Reply:
I removed this and instead use the `selectorExists` method along with the specific class name